docs/admin-get-started.md

```diff
@@ -9,102 +9,245 @@ import Tabs from '@theme/Tabs';
+    **Output**
+
+    There is no output. You can check the status of the topic in Step 5.
+
+3. Update the number of the partition to 8.
 
     **Input**
 
-    ```java
-    admin.topics().createPartitionedTopic("persistent://public/default/test-topic-1", 4);
+    ```bash
+    bin/pulsar-admin topics update-partitioned-topic \
+    persistent://public/default/test-topic-1 \
+    --partitions 8
     ```
 
-3. Update the number of the partition to 5.
+    **Output**
+
+    There is no output. You can check the number of partitions in Step 5.
+
+4. Produce some messages to the partitioned topic test-topic-1.
 
     **Input**
 
-    ```java
-    admin.topics().updatePartitionedTopic("test-topic-1", 5);
+    ```bash
+    bin/pulsar-perf produce -u pulsar://localhost:6650 -r 1000 -i 1000 persistent://public/default/test-topic-1
     ```
 
-4. Produce some messages to the topic _test-topic-1_.
+    **Output**
+
+    ```bash
+    2023-03-07T15:33:56,832+0800 [main] INFO  org.apache.pulsar.testclient.PerformanceProducer - Starting Pulsar perf producer with config: {
+      "confFile" : "/Users/yu/apache-pulsar-2.11.0/conf/client.conf",
+      "serviceURL" : "pulsar://localhost:6650",
+      "authPluginClassName" : "",
+      "authParams" : "",
+      "tlsTrustCertsFilePath" : "",
+      "tlsAllowInsecureConnection" : false,
+      "tlsHostnameVerificationEnable" : false,
+      "maxConnections" : 1,
+      "statsIntervalSeconds" : 1000,
+      "ioThreads" : 1,
+      "enableBusyWait" : false,
+      "listenerName" : null,
+      "listenerThreads" : 1,
+      "maxLookupRequest" : 50000,
+      "topics" : [ "persistent://public/default/test-topic-1" ],
+      "numTestThreads" : 1,
+      "msgRate" : 1000,
+      "msgSize" : 1024,
+      "numTopics" : 1,
+    "numProducers" : 1,
+      "separator" : "-",
+      "sendTimeout" : 0,
+      "producerName" : null,
+      "adminURL" : "http://localhost:8080/",
+
+    ...
+
+    2023-03-07T15:35:03,769+0800 [Thread-0] INFO  org.apache.pulsar.testclient.PerformanceProducer - Aggregated latency stats --- Latency: mean:   8.931 ms - med:   3.775 - 95pct:  32.144 - 99pct:  98.432 - 99.9pct: 216.088 - 99.99pct: 304.807 - 99.999pct: 349.391 - Max: 351.235
+    ```
+
+5. Check the internal stats of the partitioned topic _test-topic-1_.
 
     **Input**
 
-    ```java
-    PulsarClient client = PulsarClient.builder()
-        .serviceUrl("pulsar://localhost:6650")
-        .build();
+    ```bash
+    bin/pulsar-admin topics partitioned-stats-internal \
+    persistent://public/default/test-topic-1
+    ```
 
-    Producer<String> producer = client.newProducer(Schema.STRING)
-        .topic(topic)
-        .blockIfQueueFull(true)
-        .create();
+    **Output**
 
-    for (int i = 0; i < 100; ++i) {
-        producer.newMessage().value("test").send();
-    }
-    producer.close();
-    client.close();
+    Below is a part of the output. For detailed explanations of topic stats, see Pulsar statistics.
```

Comment:
I would suggest that we can add a link to Pulsar statistics.

Reply:
Currently, we can not add a link here. I've explained the reason in the comment above https://github.com/apache/pulsar-site/pull/504#issuecomment-1498946786